### PR TITLE
Added thruster timeout

### PR DESCRIPTION
**Pull request by dcrossman**

### CODE REVIEW

File: src/thruster.cpp [hunk 1]
```diff
@@ -180,7 +180,8 @@ void Thruster::Update()
     // TODO: (If needed): Optimize this section so that the force is only
     // calculated once per thruster per message instead of once per frame per
     // thruster.
-    if(last_thruster_msg.data.size() == 8)
+    if(last_thruster_msg.data.size() == 8 &&
+            ros::Time::now() - last_msg_receive_time < ros::Duration(0.5))
```
Comment (bkallaher) [inline src/thruster.cpp]: Maybe make the timeout duration a named constant or a parameter? That way a new Duration object isn't being created every time this line gets run.

**Review by bkallaher — CHANGES_REQUESTED** [review]
See my one comment.

**Comment by bkallaher** [comment]
You also seem to be a bit behind master.

**Comment by dcrossman** [comment]
Ok branch up to date and timeout is now loaded from param!

**Review by skallaher — APPROVED** [review]
Looks good to me!

**Review by bkallaher — APPROVED** [review]

**Pull request merged**